manim/mobject/mobject.py

```diff
@@ -1119,6 +1120,19 @@ def throw_error_if_no_points(self):
                       "for a Mobject with no points"
             caller_name = sys._getframe(1).f_code.co_name
             raise Exception(message.format(caller_name))
+    
+    # About z-index
+    def set_z_index(self, z_index_value):
```

Comment:
Docs plz (see the Wiki for guidelines)

Reply:
You can check, this is the first time I write strict DocString, not too familiar.